### PR TITLE
Feature 3 fetching data

### PR DESCRIPTION
**Pull request by henokkhm**
This Pull Request adds the necessary logic to fetch a random quotation from a free (API)[https://api-ninjas.com/api/quotes].

**Summary**
- Add useFetch **custom hook**
- Add .env to .gitignore
- Add Quote component
- Add styling to the **Quote** component

### CODE REVIEW

File: src/components/Quote.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <div className={styles.wrapper}>
+      <p className={styles.quote}>{first.quote}</p>
+      <p className={styles.right}>
+        by
+        {' '}
+        <span className={styles.author}>{first.author}</span>
+      </p>
+    </div>
+  );
```
Comment (Qoosim) [inline src/components/Quote.js:38]: - [ ] Great work so far. the `quote` property is declared to be undefined when I fired up your app. I tried to console log the `first` variable on line 27 to see the data inside it but seemed to be undefined. I created .env file and added your key to the file but still declared the `quote` as undefined. Kindly look at it and fix where the error is.

### Error
app-error

### Error
env

### Error
console

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @henokkhm, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by henokkhm** [comment]
@Qoosim I cannot reproduce that error on my machine. I double-checked that we are both on the latest commit. However, one clue that I see on the screenshot above is that the request to the API sent a status code of 400. Maybe there is a helpful message there, so can you please go to the Network tab in Developer Tools and check. Thanks!
image

**Comment by henokkhm** [comment]
Further, when I purposely put in an invalid API key in **.env**, I got a 400 status code with the following error message.
image

One thing I noticed while working on this project is that the **.env** file is loaded when I run `npm run start`, and any changes I make after that **will not have any effect until I stop the server and run** `npm run start` **again**. Maybe you are getting this error because you started the app before you saved my API key in the **.env** file.

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @henokkhm  👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ quote rendered on the HTML using the API call
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Qoosim** [comment]
> Further, when I purposely put in an invalid API key in **.env**, I got a 400 status code with the following error message. image
> 
> One thing I noticed while working on this project is that the **.env** file is loaded when I run `npm run start`, and any changes I make after that **will not have any effect until I stop the server and run** `npm run start` **again**. Maybe you are getting this error because you started the app before you saved my API key in the **.env** file.

Hi @henokkhm , I understand that I need to stop and restart the server every time I make changes while using key from .env file. However number of times I did, I got the same error message. I used the same api-key you shared in the PR inside the `.env`. 

This is what I got again.
invalid-key

**Comment by henokkhm** [comment]
@Qoosim In that case,  I may have made a mistake when I copied the API key to the submission form.

**Pull request merged**